app/models/server/definition/IServerEventsRepository.ts

```diff
@@ -0,0 +1,11 @@
+import { IServerEvent } from '../../../../definition/IServerEvent';
+
+export interface IServerEventsRepository {
```

Comment:
@sampaiodiego I already removed the interface, but it was to invert the dependency, this way our code depends on the interface rather than the class type. But I think we need to discuss more this approach